src/AdminModule/Components/Facilities/Components/AddFacilities/AddFacilities.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { TextField, Box } from "@mui/material";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { baseUrl } from "../../../../../Constants/Components/Urls";
+import {useForm} from 'react-hook-form';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import Typography from '@mui/material/Typography';
+import { toast } from "react-toastify";
+
+export default function AddFacilities({getFacilitiesList,CloseAddModal}) {
+    //useState=====>Add
+  const [openAdd, setOpenAdd] = React.useState(true);
+//   const handleOpenAdd = () => setOpenAdd(true);
+  const handleCloseAdd = () => setOpenAdd(false);
+
+  //style el two modal
+  const style = {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 400,
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 4,
+  };
+
+  //hookForm
+  const{register,handleSubmit,formState:{errors}}=useForm();
+  
+
+
+//function elllllllllll add
+async function onSubmitAddFacilities(data:object) {
+    try {
+      const response = await axios.post(`${baseUrl}/admin/room-facilities?5`,data, {
+        headers: {
+          Authorization:
+            "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NjBmNzU5ODZlYmJiZWZiYzE5ZWEyMmUiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcxMzMxNDk2NywiZXhwIjoxNzE0NTI0NTY3fQ.hZHGyq8URhmMYQ11qie8VUDRyU1JY9LujY8j7_XIamY",
+        },
+        
+      });
+      toast.success('Add is Successuflly');
+      
+      CloseAddModal();
+    handleCloseAdd();
+    getFacilitiesList();
+     
+     // console.log(data.message)
+    } catch (error) {
+      toast.error(error.message);
+     // console.log(error.message);
+    }
+  }
+  return (
+   <>
+{/*model elllllllllllllll Add------------------------------*/}
+<div>
+      <Modal
+        keepMounted
+        open={openAdd}
+        onClose={()=>{handleCloseAdd;CloseAddModal();}}
```

Comment:
we need to call the the function `handleCloseAdd()` currently we are not calling it.
It is also unclear for me why do we have two functions having a similar. It feels that both of them are doing the same functionality.